gateways/checkoutcom/request_builder.go

```diff
@@ -38,6 +38,7 @@ func buildChargeParams(authRequest *sleet.AuthorizationRequest) (*payments.Reque
 			Email: common.SafeStr(authRequest.BillingAddress.Email),
 			Name:  authRequest.CreditCard.FirstName + " " + authRequest.CreditCard.LastName,
 		},
+		ProcessingChannelId: common.SafeStr(processingChannelId),
```

Comment:
This will send nil pointer as empty string instead of leaving the field as missing, tests probably confirm if this is fine but something to be aware of

Reply:
Yeah good catch- in their json conversion, the field is marked as omitempty, so should be ok.